src/meta-srv/src/procedure/tests.rs

```diff
@@ -393,7 +393,7 @@ async fn test_submit_alter_region_requests() {
     });
 
     let status = procedure.submit_alter_region_requests().await.unwrap();
-    assert!(matches!(status, Status::Done));
+    assert!(matches!(status, Status::Executing { persist: true }));
 
     handle.await.unwrap();
 
```

Comment:
I guess that we need to add not only this test but also a integration test.
```sql
create table ~ 
alter table ~ add column foo string not null 
select * from ~
```

Reply:
Let’s add these tests once the `sqlness` test is available.

Reply:
@WenyXu `sqlness` test is available now, do you have time to add this test?

Reply:
@evenyag  @WenyXu 
I'd like to work on this if you don't mind.

Reply:
@NiwakaDev Thanks 🙏